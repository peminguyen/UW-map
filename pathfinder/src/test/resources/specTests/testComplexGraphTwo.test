CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C
AddNode graph D
AddNode graph E
AddNode graph F
AddNode graph G

AddEdge graph A B 5
AddEdge graph B A 5
AddEdge graph A C 3
AddEdge graph C A 3
AddEdge graph B C 6
AddEdge graph C B 6
AddEdge graph B D 2
AddEdge graph D B 2
AddEdge graph C D 4
AddEdge graph D C 4
AddEdge graph B E 7
AddEdge graph E B 7
AddEdge graph D E 3
AddEdge graph E D 3
AddEdge graph D F 6
AddEdge graph F D 6
AddEdge graph C F 8
AddEdge graph F C 8
AddEdge graph E F 5
AddEdge graph F E 5
AddEdge graph E G 5
AddEdge graph G E 5
AddEdge graph F G 4
AddEdge graph G F 4

FindPath graph A G