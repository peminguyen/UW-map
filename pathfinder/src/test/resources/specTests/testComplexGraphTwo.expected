created graph graph

added node A to graph
added node B to graph
added node C to graph
added node D to graph
added node E to graph
added node F to graph
added node G to graph

added edge 5.000 from A to B in graph
added edge 5.000 from B to A in graph
added edge 3.000 from A to C in graph
added edge 3.000 from C to A in graph
added edge 6.000 from B to C in graph
added edge 6.000 from C to B in graph
added edge 2.000 from B to D in graph
added edge 2.000 from D to B in graph
added edge 4.000 from C to D in graph
added edge 4.000 from D to C in graph
added edge 7.000 from B to E in graph
added edge 7.000 from E to B in graph
added edge 3.000 from D to E in graph
added edge 3.000 from E to D in graph
added edge 6.000 from D to F in graph
added edge 6.000 from F to D in graph
added edge 8.000 from C to F in graph
added edge 8.000 from F to C in graph
added edge 5.000 from E to F in graph
added edge 5.000 from F to E in graph
added edge 5.000 from E to G in graph
added edge 5.000 from G to E in graph
added edge 4.000 from F to G in graph
added edge 4.000 from G to F in graph

path from A to G:
A to C with weight 3.000
C to F with weight 8.000
F to G with weight 4.000
total cost: 15.000